src/wp-includes/default-filters.php

```diff
@@ -574,6 +574,10 @@
 add_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles' );
 add_action( 'wp_footer', 'wp_enqueue_global_styles', 1 );
 
+// SVG filters like duotone have to be loaded at the beginning of the body in both admin and the front-end.
+add_action( 'wp_body_open', 'wp_global_styles_render_svg_filters' );
+add_action( 'in_admin_header', 'wp_global_styles_render_svg_filters' );
```

Comment:
@ajlende This will load SVG filters on every admin page. We only need them on editor pages.

Reply:
I couldn't find a better place for it than `in_admin_header`. The only other hooks called in the body before the editor are the `*_admin_notices` hooks which don't make sense. At least with `get_current_screen()->is_block_editor()` it should be rendered everywhere the block editor is used by default. In cases like the site editor where the filters have to be loaded differently (see https://github.com/Mamaduka/wordpress-develop/pull/12#discussion_r809153321) it can be overridden on a case-by-case basis.